css-three-column-layout/styles.css

```diff
@@ -39,3 +39,13 @@ h1 {
   justify-content: center;
   margin: 0 auto;
 }
+
+h3 {
+  margin-bottom: 1px;
```

Comment:
This is increasing the space. We want there to be LESS space between the heading and paragraph text